File: BOJ/28235.rs
use std::io::*;

fn main() {
    let stdin = read_to_string(stdin()).unwrap();
    let word = stdin.trim();

    let output = match word {
        "SONGDO" => "HIGHSCHOOL",
        "CODE" => "MASTER",
        "2023" => "0611",
        _ => "CONTEST",
    };

    println!("{output}");
}

File: BOJ/28235.cpp
#include <iostream>
#include <unordered_map>

using namespace std;

int main() {
  ios_base::sync_with_stdio(false);
  cin.tie(nullptr);

  unordered_map<string, string> mp = {
      {"SONGDO",    "HIGHSCHOOL"},
      {"CODE",      "MASTER"},
      {"2023",      "0611"},
      {"ALGORITHM", "CONTEST"}
  };

  string s;
  cin >> s;

  cout << mp[s];

  return 0;
}